### ISSUE TITLE
can't install requirements

### ISSUE THREAD

**Issue by mircealungu** [bogdanp05/Benchmark#20]
@bogdanp05 I"m trying to install the requirements and I get errors since it does not find `perf`. can you replicate this`? 

```
ERROR: Could not find a version that satisfies the requirement perf (from -r requirements.txt (line 8)) (from versions: none)
ERROR: No matching distribution found for perf (from -r requirements.txt (line 8))

```

**Comment by bogdanp05** [comment]
Hey @mircealungu ! Thanks for pointing this out. It looks like `perf` was renamed to `pyperf`. Installing `pyperf` versions 1.6.1 or 1.70 will work. I will need to update the code to implement this renaming. Apart from this, I hope the API the package provides is still the same.

**Comment by mircealungu** [comment]
Ah, I see. Could it be that there's the same situation with performance being renamed to pyperformance?

**Comment by bogdanp05** [comment]
Yep, that's the case. Both projects are maintained by the same guy and I guess in the second half of 2019 he decided to rename.
Fixed in #21
